schedulers/argo-workflow/addons.tf

```diff
@@ -1,5 +1,5 @@
 module "eks_blueprints_kubernetes_addons" {
-  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons?ref=v4.14.0"
+  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons"
```

Comment:
Can we lock this to latest version?

Reply:
4.14 is the latest version. my change was merged but not in 4.14

Reply:
Ok, change this to 4.15. Bryant will create a new release this weekend for eks blueprints. We can merge this PR once the latest tag is available